.golangci.yml

```diff
@@ -1,81 +1,63 @@
-# This file contains all available configuration options
-# with their default values.
-
-# options for analysis running
+version: "2"
 run:
-  # default concurrency is a available CPU number
   concurrency: 16
-
-  # timeout for analysis, e.g. 30s, 5m, default is 1m
-  timeout: 5m
-
-  # exit code when at least one issue was found, default is 1
   issues-exit-code: 1
-
-  # include test files or not, default is true
   tests: true
 
-  # list of build tags, all linters use it. Default is empty list.
-  build-tags:
-
-  # which dirs to skip: they won't be analyzed;
-  # can use regexp here: generated.*, regexp is applied on full path;
-  # default value is empty list, but next dirs are always skipped independently
-  # from this option's value:
-  #     vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
-  skip-dirs:
-  # which files to skip: they will be analyzed, but issues from them
-  # won't be reported. Default value is empty list, but there is
-  # no need to include all autogenerated files, we confidently recognize
-  # autogenerated files. If it's not please let us know.
-  skip-files:
-# output configuration options
 output:
-  # colored-line-number|line-number|json|tab|checkstyle, default is "colored-line-number"
   formats:
-    - format: colored-line-number
-
-  # print lines of code with issue, default is true
-  print-issued-lines: true
-
-  # print linter name in the end of issue text, default is true
-  print-linter-name: true
+    text:
+      path: stdout
+      print-linter-name: true
+      print-issued-lines: true
 
 linters:
   enable:
-    - errcheck
     - errorlint
     - goconst
-    - gofmt
-    - goimports
-    - gosimple
-    - govet
-    - ineffassign
     - misspell
     - revive
-    - staticcheck
-    - typecheck
     - unconvert
     - unparam
```

Comment:
The migration removed linters from this list that are enabled by default. According to the output of `golangci-lint formatters` and `golangci-lint linters` the following linters/formatters are active:

```
errcheck
errorlint
goconst
gofmt
goimports
govet
ineffassign
misspell
revive
staticcheck
unconvert
unparam
unused
```